tests/utils/dockercli/containers.go

```diff
@@ -86,3 +86,24 @@ func GetVolumeProperties(volumeName, hostName string) (string, error) {
 	return ssh.InvokeCommand(hostName, cmd)
 }
 
+// WriteToContainer write data to a file in the running container
+func WriteToContainer(ip, containerName, volPath, fileName, data string) (string, error) {
```

Comment:
I will update both - something like this:

// WriteToVolume write data to a given file on given volume by starting a new container
vs.
// WriteToContainer write data to a file in the existing running container